.github/workflows/documentation.yml

```diff
@@ -16,11 +16,11 @@ jobs:
       - uses: actions/checkout@v4
       - uses: julia-actions/setup-julia@v2
         with:
-          version: '1.8'
```

Comment:
Here you've set the julia version to 1.10 but in the FastDifferentiation Project.toml you've set the Julia version to 1.9. Was there a reason to make the versions different?

Reply:
These two are independent.

The first version of Julia that supports package extensions is v1.9. Given that this package uses a package extension, it's necessary to either set the julia compat in your package's project to v1.9 or, as mentioned, to use [PackageExtensionCompat.jl](https://github.com/cjdoris/PackageExtensionCompat.jl).

The version in the documentation build is just the version of Julia used to build the documentation, there's no point in setting it to any other value than the latest supported version of Julia, which is v1.10.